c2corg_ui/static/js/auth/authservice.js

```diff
@@ -189,8 +189,18 @@ app.Authentication.prototype.hasEditRightsArticle_ = function(articleType, autho
  * @return {boolean}
  * @private
  */
-app.Authentication.prototype.hasEditRightsXreport_ = function(authorId) {
-  return this.userData.id === parseInt(authorId, 10);
+app.Authentication.prototype.hasEditRightsXreport_ = function(authorId, users) {
```

Comment:
Since outings and xreports editing rights seem to be the same, what about using ``hasEditRightsOuting_`` (could be renamed) instead?